samples/app-anonymous-users/nodejs/assets/sample.json

```diff
@@ -0,0 +1,68 @@
+[
+  {
+    "name": "officedev-microsoft-teams-samples-app-anonymous-users-nodejs",
+    "source": "officeDev",
+    "title": "Anonymous User Support",
+    "shortDescription": "This sample app shows anonymous users support in teams meeting apps.",
```

Comment:
This sample app shows anonymous users support in `Teams` meeting apps.

Reply:
Fixed